synadm/api.py

```diff
@@ -35,8 +35,6 @@
 import urllib.parse
 import time
 import re
-import click
-
```

Comment:
Leave the second empty line.

Reply:
There is no second empty line any more... the click import has already been removed (?) I double checked, there's exactly one empty line between `import re` and `class ApiRequest:`

Reply:
Please _leave_ the second blank line present. We'd like to have 2 blank lines between imports and code (like we have it between functions and methods). That's also what pep8 suggests.